package.json

```diff
@@ -1,17 +1,34 @@
 {
   "name": "eslint-config-zipscene",
-  "version": "0.0.6",
+  "version": "0.1.0",
```

Comment:
version bump for format changes?! I guess you don't like starting at `v0.0.0`. I don't care enough to change if you feel strongly.